### ISSUE TITLE
Mouse invisible on host

### ISSUE THREAD

**Issue by albionxyz** [01org/XenGT-Preview-xen#17]
Whenever I put a domU with vgt=1 with the intel driver installed my mouse cursor becomes invisible on dom0. The mouse's icon is becomes stuck on the last position. The mouse continues to work on host as normally but stays invisible until dom0 is put to foreground again.

**Comment by jikesong** [comment]
>  The mouse continues to work on host as expected but stays invisible until dom0 is set to foreground again.

When you dom0 is not the foreground one, do you have anything in dom0 visible ?

**Comment by albionxyz** [comment]
Yes, everything on dom0 is shown as expected on my second monitor exept that the mouse is functional albeit invisible.
